src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js

```diff
@@ -34,4 +52,21 @@ $(document).ready(function () {
             href: url + "{{number}}"
         });
     }
+
+    // load qTip descriptions
+    $(".groupName").hover().each(function() {
+        $(this).qtip({
+            content: {
+                text: $.trim($(this).next(".tooltiptext").text())
+            }
```

Comment:
You can add: `style: {
            classes: "qtip-shadow qtip-tipped customqtip"
        }` if you want to make it look like the commonly used style.
If we were to move one, maybe it's better to use Popper.js, since it's compatible with Bootstrap.

Reply:
Okay, we can implement popper.js in the next iteration